purchase_supplier_approve/__manifest__.py

```diff
@@ -0,0 +1,17 @@
+# Copyright 2019 RGB Consulting - Domantas Sidorenkovas
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
```

Comment:
This is a minor comment and non-blocking, but would you use single quote to all of your code? It is helpful when we do searhing for code later on.